src/main/kotlin/com/kotlindiscord/bot/extensions/HelpExtension.kt

```diff
@@ -0,0 +1,61 @@
+package com.kotlindiscord.bot.extensions
+
+import com.gitlab.kordlib.core.behavior.channel.createEmbed
+import com.kotlindiscord.bot.KDBot
+import com.kotlindiscord.bot.api.CommandInfo
+import com.kotlindiscord.bot.api.Extension
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
+
+/**
+ * Help command extension.
+ *
+ * This extension provides a `!help` command listing the available commands,
+ * along with a `!help <command>` to get more info about a specific command.
+ */
+class HelpExtension(kdBot: KDBot) : Extension(kdBot) {
+    override val name: String = "help"
+
+    override suspend fun setup() {
+        command(
+            "help",
+            aliases = arrayOf("h"),
+            signature = "!help <command>",
```

Comment:
Signature contains the prefix. The prefix used is defined in KDBot, but should probably also be configurable.